### DISCUSSION TITLE
Does "ormar(e)" really means "cabinet" in Italian?

### DISCUSSION THREAD

**Discussion by bfontaine** [collerek/ormar#282] [General]
Hello,
The README says:
> Ormar - apart form obvious ORM in name - get it's name from ormar in swedish which means snakes, and ormar(e) in italian which means cabinet.

However I can’t find any evidence that _ormare_ is anything other than a [rare verb](https://www.treccani.it/vocabolario/ormare) meaning following an animal using its footprints (_orme_).

**Comment by collerek** [comment]
Hmmm, now that you mention this I think this was supposed to be Croatian not Italian 😱 ❗ ❗ ❗ 
I'm not sure how I got this confused but thanks for pointing this out! 🤦

**Reply by collerek** [reply]
Corrected in readme
